internals/sendemail.py

```diff
@@ -118,3 +122,95 @@ def receive(bounce_message):
   message.check_initialized()
   if settings.SEND_EMAIL:
     message.send()
+
+
+def _extract_addrs(header_value):
+  """Given a message header value, return email address found there."""
+  friendly_addr_pairs = list(rfc822.AddressList(header_value))
+  return [addr for _friendly, addr in friendly_addr_pairs]
+
+
+def call_py3_task_handler(handler_path, task_dict):
+  """Request that our py3 code handle the rest of the work."""
+  handler_host = 'http://localhost:8080'
+  if settings.APP_ID == 'cr-status':
+    handler_host = 'https://chromestatus.com'
+  if settings.APP_ID == 'cr-status-staging':
+    handler_host = 'https://cr-status-staging.appspot.com'
+  handler_url = handler_host + handler_path
+
+  request_body = json.dumps(task_dict).encode()
+  logging.info('request_body is %r', request_body)
```

Comment:
Logging an encoded string doesn't tell you much. Maybe `task_dict`?

Reply:
Done.  The %r makes the encoded string readable, or about as readable as a logged dictionary.  But, your suggestion makes the code a bit simpler.